boards/tenstorrent/tt_blackhole/Kconfig.defconfig

```diff
@@ -3,3 +3,36 @@
 
 config FLASH_EX_OP_ENABLED
 	default y if FLASH_SPI_DW
+
+# Would be ideal if this were automatically generate upstream in Zephyr
```

Comment:
I'm going to get started on a PR for this upstream. It seems like a lot of unnecessary work to have to do this, when it could be a simple text substitution.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/89375

Reply:
Fixed! https://github.com/zephyrproject-rtos/zephyr/pull/89859